packages/dds/merge-tree/src/test/client.conflictFarm.spec.ts

```diff
@@ -23,16 +24,22 @@ interface IConflictFarmConfig extends IMergeTreeOperationRunnerConfig {
 	clients: IConfigRange;
 }
 
-const allOperations: TestOperation[] = [removeRange, annotateRange, insertAtRefPos];
+const allOperations: TestOperation[] = [
+	removeRange,
+	annotateRange,
+	insertAtRefPos,
+	obliterateRange,
+];
 
 export const debugOptions: IConflictFarmConfig = {
-	minLength: { min: 1, max: 1 },
-	clients: { min: 3, max: 3 },
-	opsPerRoundRange: { min: 1, max: 100 },
-	rounds: 1000,
+	minLength: { min: 1, max: 512 },
+	clients: { min: 1, max: 8 },
+	opsPerRoundRange: { min: 1, max: 128 },
+	rounds: 8,
 	operations: allOperations,
 	incrementalLog: true,
-	growthFunc: (input: number) => input + 1,
+	growthFunc: (input: number) => input * 2,
+	// resultsFilePostfix: `conflict-farm-with-obliterate-2.3.0.json`,
```

Comment:
i'm playing with regenerating with the new implementation to see if there are content differences. The test would fail regardless, but this makes it easier to debug, as i can look at the file diff.